spec/forms/bet_form_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+
+describe BetForm do
+  context 'when passed data is valid' do
+    it 'creates new bet' do
+      user = create(:user)
+      match = create(:football_match)
+
+      form = BetForm.new(
+        Bet.new,
+        {
+          user_id: user.id,
+          football_match_id: match.id,
+          home_team_score: 4,
+          away_team_score: 1
+        }
+      )
+
+      expect { form.save }.to change { Bet.count }.by 1
+    end
+  end
+
+  it 'requires scores to be positive' do
```

Comment:
Actually, it only checks that home_team_score has to be positive, please add also away_team_score.